src/commands/export.ts

```diff
@@ -256,6 +256,19 @@ const getProjectItems = async ({
   return validProjectItems;
 };
 
+const isFieldDescriptionAndColorAvailable = (gitHubEnterpriseServerVersion: string | undefined): boolean => {
```

Comment:
I don't think we need this anymore, because the minimum supported GHES version for exports is 3.12 - see `MINIMUM_SUPPORTED_GITHUB_ENTERPRISE_SERVER_VERSION_FOR_EXPORTS`. Exports are always available, in other words!

The field description and color is always available ❤️